code/ai/ai_profiles.cpp

```diff
@@ -699,7 +734,4 @@ void ai_profile_t::reset()
 	if (mod_supports_version(3, 7, 2)) {
 		flags.set(AI::Profile_Flags::Fix_ramming_stationary_targets_bug);
 	}
-	if (mod_supports_version(21, 4, 0)) {
-		flags.set(AI::Profile_Flags::Fixed_ship_weapon_collision);
-	}
```

Comment:
Rebase onto master, or remove this from your changeset, so that this item from a different PR isn't reverted.

Reply:
this is actually debris from a bad rebase onto master that I overlooked. Will fix.

Reply:
This change has now been excised